utils/textnorm_zh.py

```diff
@@ -35,6 +35,10 @@
 # SIL = [u'杠', u'槓']
 
 FILLER_CHARS = ['呃', '啊']
+ALLOW_ER_WRD='(儿女|儿子|儿孙|女儿|儿媳|妻儿|' \
```

Comment:
Is there a knowledge source of this list?

Reply:
Refer to some online materials and collected them by myself.